### PR TITLE
More updates

### PR DESCRIPTION
**Pull request by davidanthoff**
CC @lrennels

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,36 +19,39 @@ this model.
...
+see the code in the file `examples/main.jl`.
 
-The current exported functions include:
-```
-constructrice
-getrice
-getrice2010parameters
-getmarginal_rice_models
-getmarginaldamages_rice
-```
```
Comment (davidanthoff) [inline README.md]: @lrennels I removed these for now. I don't want to set expectations that we will keep this API, so maybe lets just not talk about it for now.

Comment (lrennels) [reply README.md]: Ok sounds good, are they used anywhere that we need to add prefixes to now?

Comment (davidanthoff) [reply README.md]: I don't think so... Probably by @FrankErrickson. I think it is still ok to export them, I just don't want to advertise that particular API too much, given that we will break it once we have a unified story for this stuff across all models.

Comment (lrennels) [reply README.md]: Ah I see ok

Comment (lrennels) [reply README.md]: Ok all looks good!

**Pull request merged**